Update monitoring-metricbeat.asciidoc (#31795)
elastic/beats: libbeat/docs/monitoring/monitoring-metricbeat.asciidoc

+
--
// tag::set-cluster-uuid[]

> Cluster UUID is necessary if you want to see beats monitoring in Kibana stack monitoring view. It will be grouped under the cluster for that uuid. To see the Beats monitoring section in Kibana if you have a cluster, you need to associate the {beatname_uc} with cluster UUID.  :

Reviewer: ```suggestion
The cluster UUID is necessary if you want to see {beats} monitoring in the {kib} stack monitoring view. The monitoring data will be grouped under the cluster for that UUID. To associate {beatname_uc} with the cluster UUID, set:
```

The text feels a little repetitive.
